### PR TITLE
French translation update for 0.1.6

### PR DESCRIPTION
**Pull request by OdyX**
Hi there,

I've just completed the fr.po translation, please merge before 0.1.6 :)

### CODE REVIEW

**Comment by piotrdz** [comment]
Can one of the admins verify this patch?

File: po/fr.po [hunk 1]
```diff
@@ -110,7 +107,7 @@ msgid "Access to solutions\\Show program \"4: Solution\" in the exercises"
 msgstr "Accès aux solutions\\Programme \"4: Solution\" dans les exercices"
 
 msgid "Add new program"
-msgstr ""
+msgstr "Ajouter un nouveau  programme"
```
Comment (krzys-h) [inline po/fr.po]: You have one whitespace too much here

File: po/fr.po [hunk 2]
```diff
@@ -815,21 +815,19 @@ msgid "Load\\Loads the selected mission"
 msgstr "Charger\\Charger la mission sélectionnée"
 
 msgid "Loading basic level settings"
-msgstr ""
+msgstr "Chargement des configurations de base du niveau"
```
Comment (krzys-h) [inline po/fr.po]: This text may be too long to fit in the loading screen

File: po/fr.po [hunk 3]
```diff
@@ -1446,9 +1438,8 @@ msgstr "Standard\\Remet les couleurs standards"
 msgid "Start"
 msgstr "Départ"
 
-#, fuzzy
 msgid "Starting..."
-msgstr "Départ"
+msgstr "Lancement…"
```
Comment (krzys-h) [inline po/fr.po]: There is some weird ... character here

**Pull request merged**